packages/toolpad-app/src/toolpad/AppEditor/PagesExplorer/index.tsx

```diff
@@ -321,11 +321,7 @@ export default function PagesExplorer({ className }: PagesExplorerProps) {
     [appStateApi, dom],
   );
 
-  const pagesTreeItemSx = {
-    '.MuiTreeItem-label': {
-      ml: -1.5,
-    },
-  };
+  const alphabeticallySortedPages = React.useMemo(() => sortBy(pages, 'name'), [pages]);
```

Comment:
That's not alphabetically sorted. e.g. `'a' < 'B' === false`.

Not a fan of `sortBy`, I'd just do:

```tsx
[..pages].sort((page1, page2) => page1.name.localeCompare(page2.name)
```

Reply:
Uh, that's right about the alphabetical ordering... And nice, I didn't know about `localeCompare` or `toSorted`.
The first approach seems fine, I tried implementing the second abstraction you proposed but I'm not really sure how to? And we might not need that abstraction yet anyway...